Station.h

```diff
@@ -34,6 +34,7 @@ class Station {
     float getY(){return cordY;}
     int getType(){return type;}
     int getProduct(){return product;}
+    int getMaxProfuct(){return maxProduct;}
```

Comment:
может все-таки Product
```suggestion
    int getMaxProduct(){return maxProduct;}
```

Reply:
я думаю все таки да